### PR TITLE
Adding PCI PaaS Reference Architecture Solution

### PR DESCRIPTION
**Pull request by consultguru**
# **Adding PCI PaaS Reference Architecture Solution**

Microsoft Sponsors are

-  Azure Security & Compliance Team: Alan Ross  

- Azure Operations Management Team: Eric Winner

- Azure Marketplace Partner Solutions: Walter Harp.

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@consultguru__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

File: pci-paas-webapp-ase-sqldb-appgateway-keyvault-oms/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,795 @@
+﻿{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "_artifactsLocation": {
+      "type": "string",
+      "defaultValue": "https://raw.githubusercontent.com/AvyanConsultingCorp/pci-paas-webapp-ase-sqldb-appgateway-keyvault-oms/master",
```
Comment (singhkays) [inline pci-paas-webapp-ase-sqldb-appgateway-keyvault-oms/azuredeploy.json]: URL should point to Azure repo

File: pci-paas-webapp-ase-sqldb-appgateway-keyvault-oms/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,795 @@
...
+        "description": "The password to use for the bastion host VM administrator."
+      }
+    },
+    "sqlAdministratorLoginPassword": {
+      "type": "securestring",
+      "metadata": {
+        "description": "The password to use for the database server administrator."
+      }
+    },
+    "SqlNotificationEmailAddress": {
```
Comment (singhkays) [inline pci-paas-webapp-ase-sqldb-appgateway-keyvault-oms/azuredeploy.json]: All parameter names need to follow camelCase

File: pci-paas-webapp-ase-sqldb-appgateway-keyvault-oms/azuredeploy.json [hunk 3]
```diff
@@ -0,0 +1,795 @@
...
+      }
+    },
+    "customHostName": {
+      "type": "string",
+      "defaultValue": "azurepcisamples.com",
+      "metadata": {
+        "description": "Provide the Custom Host Name."
+      }
+    },
+    "AzureADApplicationClientID": {
```
Comment (singhkays) [inline pci-paas-webapp-ase-sqldb-appgateway-keyvault-oms/azuredeploy.json]: camelCase here as well and other parameters below as well

File: pci-paas-webapp-ase-sqldb-appgateway-keyvault-oms/azuredeploy.json [hunk 4]
```diff
@@ -0,0 +1,795 @@
...
+      "metadata": {
+        "description": "The AD User password to use for the application's connections to the database server."
+      }
+    }
+
+  },
+  "variables": {
+    "storageAccountDiagnostics": "[concat('stgdiag', uniquestring(resourceGroup().id))]",
+    "storageAccountPreRelease": "stgpcipaasreleases",
+    "WebContainerName": "pci-paas-web-container",
```
Comment (singhkays) [inline pci-paas-webapp-ase-sqldb-appgateway-keyvault-oms/azuredeploy.json]: camelCase for variables as well

**Pull request merged**